core/src/main/java/org/apache/carbondata/core/index/TableIndex.java

```diff
@@ -591,14 +591,14 @@ public void onEvent(Event event, OperationContext opContext) {
    * @throws IOException
    */
   public Map<String, Long> getBlockRowCount(List<Segment> allsegments,
-      final List<PartitionSpec> partitions, TableIndex defaultIndex)
+      final List<PartitionSpec> partitions, TableIndex defaultIndex, boolean isIUDFlow)
```

Comment:
It is not good idea to add such parameter, which hides the actual behavior of this function

Can you refactor it

Reply:
done